cobra/core/model.py

```diff
@@ -500,17 +505,34 @@ def add_boundary(self, metabolite, type="exchange", reaction_id=None,
         >>> demand.build_reaction_string()
         'atp_c --> '
         """
-        types = dict(exchange=("EX", -ub, ub), demand=("DM", 0, ub),
-                     sink=("SK", -ub, ub))
+        if ub is None:
+            ub = CONFIGURATION.upper_bound
+        if lb is None:
+            lb = CONFIGURATION.lower_bound
+        types = {
+            "exchange": ("EX", lb, ub, "SBO:0000627"),
```

Comment:
I would prefer taking the SBO terms from `cobra.medium.boundary_types.sbo_terms`. Might make it a bit uglier but future proof since there is a single source of truth for the SBO terms (for instance if they change at some point). So fir instance use `sbo_terms["exchange"]` rather than `"SBO:0000627"`.

Reply:
I anyway want to make arguing about the ontology possible in future so you're absolutely right.